python_modules/dagster-graphql/dagster_graphql_tests/graphql/snapshots/snap_test_graph.py

```diff
@@ -7,6 +7,111 @@
 
 snapshots = Snapshot()
 
+snapshots['TestGraphs.test_basic_graphs[non_launchable_postgres_instance_lazy_repository] 1'] = {
```

Comment:
I guess these snapshot changes are ok since they are the result of a rename + are purely additive